diff --git a/src/vs/workbench/contrib/browserView/electron-browser/features/browserTabManagementFeatures.ts b/src/vs/workbench/contrib/browserView/electron-browser/features/browserTabManagementFeatures.ts
index 8c0dfd2..0e44460 100644
--- a/src/vs/workbench/contrib/browserView/electron-browser/features/browserTabManagementFeatures.ts
+++ b/src/vs/workbench/contrib/browserView/electron-browser/features/browserTabManagementFeatures.ts
@@ -485,19 +485,6 @@ class OpenOrListBrowsersAction extends Action2 {
 				weight: KeybindingWeight.WorkbenchContrib,
 				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.Slash,
 			},
-			menu: {
-				id: MenuId.TitleBar,
-				group: 'navigation',
-				order: 10,
-				when: ContextKeyExpr.and(
-					ContextKeyExpr.equals('config.workbench.browser.showInTitleBar', false).negate(),
-					ContextKeyExpr.or(
-						CONTEXT_BROWSER_EDITOR_OPEN,
-						// This is a hack to work around `true` just testing for truthiness of the key. It works since `1 == true` in JS.
-						ContextKeyExpr.equals('config.workbench.browser.showInTitleBar', 1)
-					)
-				),
-			}
 		});
 	}
 
